### DISCUSSION TITLE
Why are some dependencies red?

### DISCUSSION THREAD

**Discussion by estuday** [python-poetry/poetry#9645] [Q&A]
Hi,
I used poetry show and it showed that the vllm package was red.
![image](https://github.com/user-attachments/assets/35850d67-efde-43f7-9717-cdf2266ca2e6)
Many people told me that they need to update, but after typing the following command, it shows that no package needs to be updated
![image](https://github.com/user-attachments/assets/9895e3f1-5454-405d-9185-d9e44edab4c4)
how can i solve it?

**Comment by radoering** [comment]
See https://github.com/orgs/python-poetry/discussions/6330 for the meaning of the colors.

I suppose the package is not installed. Either it is not required for your environment (e.g. only for another Python version or platform) or you can run `poetry install` to install it.
